brands/amenity/pharmacy.json

```diff
@@ -1810,4 +1822,4 @@
       "name": "屈臣氏"
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
if you run npm run build this should go away as well. We need an editor config file to prevent these getting added back in every time.

Reply:
I don't actually know javascript and have just been hacking these together in a browser.  I just tried to manually remove that bracket but that caused it to fail a check.  How should I proceed?